Sources/SE0000_KeyPathReflection/Reflection.swift

```diff
@@ -10,6 +10,9 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Swift
+import Foundation
```

Comment:
This import worries me because if the idea is to ultimately move this into the standard library, I don't believe the stdlib can import foundation iirc. That might be wrong though, but it looks like most of the need for this is in handling `StaticString` and creating strings whose backing storage is the pointer to the field names rather than copying the contents. I see another use for `strlen`, but again that's only for `StaticString`. I don't know if this comment will come before or after my comment on the use of `StaticString`, but refer to that comment which might alleviate this import.

Reply:
I imported Foundation here because I used `init(bytesNoCopy:length:encoding:freeWhenDone:)` to create `String`s from `StaticString`s in `allNamedKeyPaths<T>(for: T)` to avoid a copy.  I was aware that importing Foundation would not be possible in the standard library, but I thought it may become trivial to create a no-copy `String` when all of this lands in the standard library (we would have access to `String`'s internals, but I'm not familiar).  I've reverted this, as well as `StaticString`.